torch/_inductor/scheduler.py

```diff
@@ -922,9 +922,22 @@ def can_fuse_vertical(self, node1, node2):
         be scheduled before the fusion of node1 and node2.
         """
         node1_names = node1.get_names()
-        remaining_deps = {
-            dep.name for dep in node2.unmet_dependencies - node1.read_writes.writes
-        }
+        computed_deps = set()
+        for rd in node2.unmet_dependencies:
```

Comment:
Could we normalize this some other way? 

Like, the logic here seems like something that's generically true about `MemoryDep`?

Reply:
I'll leave that for the future, we could create a comparison op for MemoryDeps that would return True for this.